tests/appsec/iast/_test_iast_fixtures.py

```diff
@@ -101,14 +101,16 @@ def test_telemetry_metric_instrumented_sink(self):
         assert series, f"Got no series for metric {expected_metric}"
         logging.debug("Series: %s", series)
         expected_tag = f"vulnerability_type:{self.vulnerability_type}"
-        series = [s for s in series if expected_tag in s["tags"]]
+        series = [s for s in series if expected_tag in s["tags"]] or [
+            s for s in series if expected_tag.lower() in s["tags"]
+        ]
         assert series, f"Got no series for metric {expected_metric} with tag {expected_tag}"
         for s in series:
             assert s["_computed_namespace"] == expected_namespace
             assert s["metric"] == expected_metric
             assert s["common"] is True
             assert s["type"] == "count"
-            assert set(s["tags"]) == {expected_tag}
+            assert set([tag.lower() for tag in s["tags"]]) == {expected_tag.lower()}
```

Comment:
You are checking that `s["tags"]` is a list containing only `expected_tag` (ok if there are several occurence of it). Is it what you want to check ?

Reply:
No, the fix adds a case insensitive validation of the tags. Java sends tags like "vulnerability:WEAK_RANDOMNESS" and python tracer transforms tags to lowercase "vulnerability:weak_randomness"